samples/kubernetes/cloud-config/master-config-template.yaml

```diff
@@ -1,179 +0,0 @@
-#cloud-config
```

Comment:
I think I agree with removing all the samples from this repo, but we should think about where the belong if we want to keep maintaining them (they are used by some of our guides)